Observatory-Control-System/BLOCKS/block-jira-project-workflow.rst

```diff
@@ -0,0 +1,245 @@
+.. |author| replace:: *Erik Dennihy*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *none*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+.. _BLOCK-Jira-Workflow:
+
+###################
+BLOCK JIRA Workflow
+###################
+
+The Rubin Observatory uses observing blocks to design, plan, execute, and track tasks at the summit which require data taking.
+These tasks range from pure engineering (e.g. closed-dome TMA velocity tests) to science-driven surveys. 
+In most cases, the planning and execution of such tasks must take place on a much shorter timescale than the analysis of the data that is collected, necessitating a separate JIRA project and workflow. 
+This workflow is captured in the BLOCK Project, and described in detail here. 
+
+.. figure:: _static/BLOCK_Jira_workflow.png
+    :name: BLOCK Jira Project Workflow
+    :align: center
+    :width: 480
+
+    A block diagram of the workflow contained in the BLOCK Jira project.
+
+BLOCK Ticket Roles and Workflow
+===============================
+
+BLOCK tickets are heavily relied on during both the task planning and task execution phases, 
+and provide a central point of communication between stakeholders and the summit crew. 
+Since nearly all groups across the observatory are stakeholders in the tests that are carried out at the summit, clear roles and expectations must be defined for how the JIRA BLOCK tickets are used. 
+Here we define the various roles and their expectations, and present a brief overview of the BLOCK Ticket workflow. 
+
+
+Roles and Responsibilities
+--------------------------
+In the JIRA issues, there are only two formal roles defined: Proposer and Assignee. 
+
+**Proposer**
+^^^^^^^^^^^^
+The Proposer is the stakeholder requesting the observing task. 
+They are chiefly responsible for the initial ticket submission, including providing links to supporting materials, and the final ticket sign-off (see Transitioning to Done). 
+They are expected to work closely with the Assignee during the Preparation phase, but are not expected to be involved in planning when a task will take place or during the task execution (unless otherwise required). 
+Anyone can take the role of Proposer as long as they are knowledgeable in the task requirements and completion criteria. 
```

Comment:
In line with Tiago's comment, I think this also should be given in bullets. I've expanded it a little to include a couple of extra information. Below, I copied what I did for the "proposer" and "assignee" responsibilities. Feel free to use (or not use) it:

```
Proposer
--------
The Proposer is typically a stakeholder interested in the observing task's outcomes.
Key responsibilities include:

- **Initial Ticket Submission**:
  - They are chiefly responsible for the initial ticket submission, providing a detailed description of the observing task.
  - Includes necessary supporting materials, such as links to relevant tickets (e.g., SITCOM, LVV) or documentation on Confluence pages.

- **Collaboration during Preparation**:
  - Collaborates with the Assignee during the Preparation phase to ensure all task requirements and completion criteria are clearly understood.

- **Final Verification and Sign-Off**:
  - Review the outcomes after task execution to ensure objectives have been met.
  - Verifies and confirms task completion, reviewing any related data or results.
  - Transitions the ticket to "Done" to officially close the issue once satisfied with the task completion.

   .. note::
      Anyone can take the role of Proposer as long as they are knowledgeable in the task requirements and completion criteria.

Assignee
--------
The Assignee is a member of the Commissioning Science team, responsible for overseeing the task from preparation through execution.
Responsibilities include:


- **Task Preparation and Planning**:
  - Collaborates with the Proposer to understand task requirements during early preparation stages.
  - Prepares the task for execution, including finalizing methodological details and setting up necessary components.

- **Marking as Ready**:
  - Marks the ticket as "Ready" once all preparations are complete and validated, indicating it is prepared for execution.

- **Advocacy and Tracking**:
  - Advocates for the task in planning meetings to ensure it is prioritized and scheduled.
  - Tracks the task’s progress through to execution, addressing any issues that may arise.

- **Role during Execution**:
  - Ensures everything is in place for successful execution, although the actual execution is typically done by an Observing Specialist.

```

Reply:
Thanks, I like this format. I will adopt.